openfold/data/data_pipeline.py

```diff
@@ -735,22 +736,11 @@ def read_msa(start, size):
 
             fp.close()
         else:
-            for f in os.listdir(alignment_dir):
-                path = os.path.join(alignment_dir, f)
-                filename, ext = os.path.splitext(f)
-
-                if(ext == ".a3m"):
-                    with open(path, "r") as fp:
-                        msa = parsers.parse_a3m(fp.read())
-                elif(ext == ".sto" and filename not in ["uniprot_hits", "hmm_output"]):
-                    with open(path, "r") as fp:
-                        msa = parsers.parse_stockholm(
-                            fp.read()
-                        )
-                else:
-                    continue
-
-                msa_data[f] = msa
+            # Now will split the following steps into multiple processes 
```

Comment:
If we already generated the pkl file, then we should check that it exists before re-parsing the msas. Or does it get removed somewhere?

Reply:
Oh also, is there reason we couldn't just call a function to do this instead of running the script with subprocess?